lib/msal-browser/src/custom_auth/core/error/MsalCustomAuthError.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Constants } from "@azure/msal-common/browser";
+import { CustomAuthError } from "./CustomAuthError.js";
+
+export class MsalCustomAuthError extends CustomAuthError {
+    subError: string | undefined;
+
+    constructor(
+        error: string,
+        errorDescription?: string,
+        subError?: string,
+        correlationId?: string
+    ) {
+        super(error, errorDescription, correlationId);
+        Object.setPrototypeOf(this, MsalCustomAuthError.prototype);
+
+        this.subError = subError || Constants.EMPTY_STRING;
```

Comment:
Constants.EMPTY_STRING is deprecated (removed in v5 branch), please use empty quotes instead
```suggestion
        this.subError = subError || "";
```

Reply:
Sure, fixed. Just curious, why is it deprecated? Is it because for reducing the package size?

Reply:
Yes, it does not get minified in the output bundle